cherry_picker/flit.ini

```diff
@@ -2,6 +2,8 @@
 module = cherry_picker
 author = Mariatta Wijaya
 author-email = mariatta.wijaya@gmail.com
+maintainer = Python Core Developers
+maintainer-email = core-workflow@mail.python.org
```

Comment:
Thanks :) Removed `mail.`. It was copied from [blurb](https://github.com/python/core-workflow/blob/master/blurb/flit.ini#L6) but I'll leave that alone for now :)